main/tex_compiler/tex_compiler.py

```diff
@@ -96,10 +98,9 @@ def _beskrivningfordelning_fordelning_sum(self, sales):
 
     def _prepend_commands(self):
         cmds = str(self.tex_commands)
+        with open(self.intakt_skeleton, "r") as skel_f:
+            tex_skeleton = skel_f.read()
```

Comment:
An alternative way to read (and write) files with pathlib:
```suggestion
        tex_skeleton = self.intakt_skeleton.read_text()
```
Which do you think is better?

(If we decide to use read_text and write_text maybe it is better to change it everywhere in a separate pull request)

Reply:
I agree that it looks better, might be better to open a new issue and do it in another PR.

Reply:
I opened #22 for this containing this information.